styles/mobile-first-style.css

```diff
@@ -167,3 +167,350 @@ section {
+  line-height: 44px;
+}
+
+section .card-description {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.card-description > ul {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin-left: 0;
+  padding-left: 0;
+}
+
+.card-description > ul li {
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+}
+
+.card-list-title {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  margin-left: 40px;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  list-style-type: none;
+}
+
+.Job-title_and_year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  margin-left: 12px;
+}
+
+.notice {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  margin-left: 40px;
+  padding-right: 16px;
+}
+
+.first-card {
+  margin-top: 114px;
+}
+
+.card {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding: 16px;
+  width: (100% - 40%);
+  margin-left: 24px;
+  margin-right: 24px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  -webkit-box-flex: 0;
+  -ms-flex: none;
+  flex: none;
+  -webkit-box-ordinal-group: 1;
+  -ms-flex-order: 0;
+  order: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+}
+
+.cards {
+  margin-top: 15px;
+}
+
+.grid-container {
+  display: -ms-grid;
+  display: grid;
+  -ms-grid-columns: auto;
+  grid-template-columns: auto;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  gap: 88px;
+}
+
+.programming-languages {
+  margin-left: 0;
+  padding-left: 26px;
+  list-style-type: none;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  justify-content: center;
+}
+
+.project-button,
+.resume-button {
+  background-color: #fff;
+  color: #6070ff;
+  font-weight: 500;
+  margin-top: 20px;
+  margin-left: 40px;
+  width: 35%;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.resume-button {
+  margin-left: 24px;
+}
+
+.about-me {
+  background: #fff;
+  margin-top: 88px;
+  border-top-right-radius: 80px;
+}
+
+.btn:hover {
+  cursor: pointer;
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.languages {
+  margin-top: 30px;
+  margin-left: 24px;
+}
```

Comment:
- [ ] Kindly use the appropriate margin like in the design in the Figma template

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212113850-71977aec-621f-4777-9cdd-0dd2a9d5076d.png)   |   ![image](https://user-images.githubusercontent.com/83360636/212113713-d2ccc933-2ff5-46a6-820a-8afa832ab2a3.png)    |